src/Jackett.Common/Indexers/AwesomeHD.cs

```diff
@@ -0,0 +1,188 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+using Jackett.Common.Models;
+using Jackett.Common.Models.IndexerConfig;
+using Jackett.Common.Services.Interfaces;
+using Jackett.Common.Utils;
+using Newtonsoft.Json.Linq;
+using NLog;
+
+namespace Jackett.Common.Indexers
+{
+    public class AwesomeHD : BaseWebIndexer
+    {
+        private static string SearchUrl => "searchapi.php";
+        private static string TorrentUrl => "torrents.php";
+        private new ConfigurationDataPasskey configData => (ConfigurationDataPasskey)base.configData;
+
+        public AwesomeHD(IIndexerConfigurationService configService, Utils.Clients.WebClient c, Logger l, IProtectionService ps)
+            : base("Awesome-HD",
+                description: "An HD tracker",
+                link: "https://awesome-hd.me/",
+                caps: new TorznabCapabilities(),
+                configService: configService,
+                client: c,
+                logger: l,
+                p: ps,
+                configData: new ConfigurationDataPasskey())
```

Comment:
An info block on the config to inform users where to get their passkey is also usual for non-standard login configs.

Reply:
added